### DISCUSSION TITLE
Will future language bindings replace or extend Rust?

### DISCUSSION THREAD

**Discussion by alinnert** [tauri-apps/tauri#3532] [Q&A] [answered]
A quick question out of curiosity: Will the planned language bindings for Deno, Go, etc. allow a language to entirely replace Rust or will I be required to install Rust in order to build a Tauri app nonetheless because there's still some core code written in Rust?

**Comment by FabianLars** [comment, marked as answer]
You will always* have to have rust installed to use tauri, because the whole codebase is in rust.

* Unless we ever make tauri available as some kind of pre-compiled library thingy (don't wait for it).
